packages/jest-leak-detector/src/index.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
```

Comment:
you can remove this - when we use ESM the file is strict by default

Reply:
✅